woocommerce/woocommerce-ios
# Add associated domain entitlement for wordpress.com

## Description

This PR adds associated domain entitlement, so we have password autofill filter/suggestion on `wordpress.com` domain.

Docs:
- https://developer.apple.com/documentation/security/password_autofill/
- https://developer.apple.com/documentation/xcode/supporting-associated-domains
- https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_associated-domains

### Blockers
- [x] update `apple-app-site-association` by @astralbodies (p1636641097264100-slack-C6H8C3G23)
- [x] add "Associated Domains" capability on WC App IDs by @AliSoftware (pdnsEh-3L-p2)

## Test

> Apple’s Content Delivery Network requests the apple-app-site-association file for your domain within 24 hours. Devices check for updates approximately once per week after app installation.

1. Delete previous version of the app from device. App Store updates won't need this step.
2. Run local build or download installable build from comments below.
3. Make sure you see suggested profile on Log In screen.
4. Tap on "key" icon above keyboard -> choose your password manager.
5. Verify that you see filtered wp.com accounts in the list of all suggestions.
6. Complete login flow by selecting an account from suggestions.

---
Update release notes:
